R4ML/R/ml.model.lm.R

```diff
@@ -338,10 +339,127 @@ predict.r4ml.lm <- function(object, data) {
 
     # orgainze result for presentation to user
     if (testing) {
-      stats <- SparkR::as.data.frame(r4ml.read.csv(statsPath, header=FALSE, stringsAsFactors=FALSE))
-      return(list("predictions"=preds, "statistics"=stats))
+      stats <- SparkR::as.data.frame(r4ml.read.csv(statsPath, header = FALSE,
+                                                   stringsAsFactors = FALSE))
+      return(list("predictions" = preds, "statistics" = stats))
+    } else {
+      return(list("predictions" = preds))
     }
-    else {
-      return(list("predictions"=preds))
+}
+
+r4ml.lm.se <- function(object, seed = 5) {
+  logSource <- "r4ml.lm.se"
+
+  df <- object@training_data
+  training_nrow <- SparkR::nrow(df)
+
+  if (training_nrow > 100000) {
```

Comment:
Pl explained why we are putting 100k . i.e "Since the standard deviation will be same for sample and population we will estimates population s.e via the sample s.e., the 100k is based on the how much data we can reasonably fit in the driver mem"